python/cudf/cudf/core/column/interval.py

```diff
@@ -110,3 +113,10 @@ def as_interval_column(self, dtype, **kwargs):
             )
         else:
             raise ValueError("dtype must be IntervalDtype")
+
+    def to_pandas(
+        self, index: ColumnLike = None, nullable: bool = False, **kwargs
+    ) -> "pd.Series":
+        return pd.Series(
+            pd.IntervalDtype().__from_arrow__(self.to_arrow()), index=index
+        )
```

Comment:
Is `nullable` handled?

Reply:
No. @shwina said that he and @marlenezw were unable to get other methods of converting our interval types to pandas to work because cudf/pandas/pyarrow all have different semantics for intervals and this was the only sequence of ops that succeeded, so we may be stuck with this for now. I've added a comment to that effect.